### ISSUE TITLE
Install documentation and locations are disparate

### ISSUE THREAD

**Issue by kantai** [blockstack/blockstack-browser#806]
Right now, the path to installing browser on various platforms is unclear and documented in many different locations

If you go to:
https://blockstack.org/install

You get a link to developer alphas on github, which is the releases page for browser (this repo). These releases for the most part _only_ link the releases for macOS.

Instructions for installing on windows and linux also live here:

https://github.com/blockstack/blockstack-core/#installing-blockstack

which links to here:

https://github.com/blockstack/packaging/tree/master/browser-core-docker#installing-blockstack-browser-and-api-with-docker

At this point, we should figure out 
1. Where installation steps should be documented
2. Where the installation files should live (i.e., be downloaded from)
3. How to make sure that the links at https://blockstack.org/install actually get you to the windows and linux installs too
4. Integrating the windows/linux installs into the release process

Basically, we can't figure out (4) until we've figured out the other steps.

@larrysalibra @jackzampolin

**Comment by jackzampolin** [comment]
1. I think they should live at `blockstack.org/install`. That should be download links for the 'packages' for Mac, Win and Linux. All other install instructions point users there
2. Currently the browser releases come off of GH and looks like the Win install comes off of an Azure box. I think we should setup the releases in Az Blob storage or s3 and provide links from there.
3. Once we have the packages this should be easy
4. This should be pretty quick. We already have the docker builds setup. All we need to do is peg versions in the installers and update on release. Then all we need is to make some build hooks on the `packaging` repo and have that push to our storage system. 

Does that sound right?

**Comment by larrysalibra** [comment]
How about this?

Redirect https://blockstack.org/install to https://blockstack.org/download

Have the 3 platforms listed on that page.

Instructions & download links live on:
https://blockstack.org/download/macos
https://blockstack.org/download/windows
https://blockstack.org/download/linux

(see keybase for how short the instructions should be:

https://keybase.io/docs/the_app/install_macos
https://keybase.io/docs/the_app/install_windows)

The installation files can continue to live on the github releases for the `blockstack-browser` repo.

We can programmatically pull the links for each release using their API and display the download links on the above pages. Release v0.15.0: https://api.github.com/repos/blockstack/blockstack-browser/releases/7673103


Our release process can be as follows:

1. I create the mac build & tag the release
2. I create a draft release on github and upload the mac build `.dmg` file
3. @kantai creates the linux and windows builds and adds them to the draft release
4. One of us clicks publish.
5. The website is automatically updated because it always displays download links from the latest github release.

**Comment by jackzampolin** [comment]
I like the `blockstack.org/download/<platform>` format. We could always push the new build to it's tag as well as the `<platform>/latest`. Might look something like this:

```
├── linux
│   ├── latest.tar.gz
│   ├── v0.14.5.tar.gz
│   └── v0.15.0.tar.gz
└── windows
    ├── latest.dmg
    ├── v0.14.5.dmg
    └── v0.15.0.dmg
```

**Comment by kantai** [comment]
@larrysalibra: this process sounds great, though I could be convinced by Jack to put the releases on Azure or S3.

We should see how much of Step 3 can be automated from the release tags. As @jackzampolin said, the windows installer and the linux install both use the docker `launcher` script, and so those can be set to pull from release tags on quay. The Makefile in the packaging repo is already scripted to build the windows installer inside a docker container so it can be built from pretty much anywhere. We'd just need to add the release tag to the script so that it gets hard-coded in the `launcher`

**Comment by jackzampolin** [comment]
I have a feeling this is a job for Jenkins

**Comment by larrysalibra** [comment]
> though I could be convinced by Jack to put the releases on Azure or S3.

I definitely support putting releases on a CDN in the long-term. Github serves from s3 right now and it's crazy slow. 

The process should definitely be more automated in the long-term. We can't entirely automate the mac release process right now because the code signing steps require my private keys. (i've opened an issue on operations to create an apple developer account for blockstack)

Speaking of code signing, are we signing the windows build?

**Comment by kantai** [comment]
No, we don't sign the windows installer, but we should.

**Comment by jackzampolin** [comment]
What about checksums? Could we avoid signing and streamline with that?

**Comment by kantai** [comment]
Checksums would be easier than signing the installer, but the advantage of signing the installer is that the Windows pop-ups won't scream that the origin of the installer is "Unknown"
